lib/internal/fs/streams.js

```
@@ -204,11 +214,9 @@ ReadStream.prototype._read = function(n) {
 };
 
 ReadStream.prototype._destroy = function(err, cb) {
-  const isOpen = typeof this.fd !== 'number';
-  if (isOpen) {
-    this.once('open', closeFsStream.bind(null, this, cb, err));
+  // if stream is not open it will be closed in open()
+  if (typeof this.fd !== 'number')
```

we are swallowing the original error here.